### PR TITLE
Various fixes for ConddbEssource input parameters handling

### PR DESCRIPTION
**Pull request by ggovi**
This PR include:
- A Warning message is printed when a GT ora a Tag are read from a production V1 account 
- A clean up of the handling of DB ESSource input parameters from cfg
- A fix to support '@' character(s) in sqlite file names

### CODE REVIEW

**Comment by ggovi** [comment]
+1

**Comment by diguida** [comment]
@cmsbuild 
please test

**Comment by diguida** [comment]
@cmsbuild 
please test

**Comment by diguida** [comment]
@ggovi let's see if now it works.

**Comment by ggovi** [comment]
+1

**Comment by ggovi** [comment]
@cmsbuild 
please test

File: CondCore/CondDB/src/OraDbSchema.cc [hunk 1]
```diff
@@ -311,7 +311,7 @@ namespace cond {
 
     bool OraGTTable::select( const std::string& name ){
       cond::TagCollectionRetriever gtRetriever( m_session, "", "" );
-      return gtRetriever.existsTagCollection( name+"::All" );
+      return gtRetriever.existsTagCollection( name );
```
Comment (diguida) [inline CondCore/CondDB/src/OraDbSchema.cc]: This re-adds the need for CondDBv1 GT to have the `::All` in the name passed in the configuration

**Comment by diguida** [comment]
+1

File: Configuration/AlCa/python/GlobalTag_condDBv1.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    if inputGTParams.find("_") == -1:
+        raise Exception("Invalid GT name. It does not contain an _, it cannot be used for replacements.")
+    prefix = inputGTParams.split("_")[0]
+    for i in range(0, len(mainList)):
+        if mainList[i].split("_")[0] == prefix:
+            return i
+    return -1
+
+
+from Configuration.AlCa.autoCond import aliases
```
Comment (diguida) [inline Configuration/AlCa/python/GlobalTag_condDBv1.py]: @ggovi this should be changed to

````
from Configuration.AlCa.autoCond import aliases
````

**Pull request merged**